.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "semi": true,
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
+      "options": {
+        "semi": true
+      }
+    }
+  ]
+}
```

Comment:
- No need for overrides here, as different configuration for certain file extensions, folders and specific files is not required.
- Define, at least, basic configuration, https://prettier.io/docs/en/configuration.html#basic-configuration